chain/network/src/peer/testonly.rs

```diff
@@ -292,3 +292,10 @@ impl PeerHandle {
         (outbound_stream, inbound_stream)
     }
 }
+
+pub(crate) fn make_codec() -> Codec {
+    use near_metrics::prometheus;
+    let opts = prometheus::Opts::new("test", "test");
+    let gauge = prometheus::IntGaugeVec::new(opts, &["test"]).unwrap();
+    Codec::new(gauge.with_label_values(&["127.0.0.1:3030"]))
```

Comment:
To make the test more realistic, it would be better to construct the Codec with the actual addr of the Peer.
But that will be easier once the FramedWrite construction is moved inside the PeerActor::new.

Reply:
Yeah, I think for realism we actually want to use the real metrics here, rather that this fake gauge, but that is idelly achieved by shading more code between tests & prod, rather than by duplicating said code.